.circleci/scripts/binary_upload.sh

```diff
@@ -23,12 +25,11 @@ if [[ "${DRY_RUN}" = "disabled" ]]; then
   AWS_S3_CP="aws s3 cp"
 fi
 
-# this is special build with all dependencies packaged
-if [[ ${BUILD_NAME} == *-full* ]]; then
-  UPLOAD_SUBFOLDER="${UPLOAD_SUBFOLDER}_full"
+if [[ -n "$UPLOAD_ENDPOINT" ]]; then
+  UPLOAD_BUCKET="s3://pytorch-downloads/"
```

Comment:
The S3 upload bucket is called "s3://pytorch" right? I think R2 is the only one called `pytorch-downloads`?

Reply:
hi @zxiiro this is correct this is why this is bucket is changed to pytorch-download

Reply:
Okay. The URI `s3://` was confusing me.